src/radical/pilot/agent/executing/popen.py

```diff
@@ -361,6 +361,10 @@ def _handle_task(self, task):
         self._log.info('Launching task %s via %s in %s', tid, cmdline, sbox)
 
         _launch_out_h = ru.ru_open('%s/%s.launch.out' % (sbox, tid), 'w')
+        # `start_new_session=True` is default, which enables decoupling
+        # from the parent process group (part of the task cancellation)
+        _start_new_session = self._cfg['resource_cfg'].\
```

Comment:
This actually does not need a leading underscore since it is scoped to the method, right?  But I see we did that also for other vars here, so I guess that works...

Reply:
yeah, this is just to keep it in sync with other variables names, and specifically with one above